lib/pxutil.js

```diff
@@ -264,6 +266,29 @@ function getTokenObject(cookie, delimiter = ':') {
     return { key: '_px3', value: cookie };
 }
 
+function getGraphqlData(req) {
+    if (!req.path.includes('graphql')) {
+        return null;
+    }
+
+    const queryArray = req.body['query'] ? req.body['query'].split(/[^A-Za-z0-9_]/) : null;
+    if (!queryArray || queryArray.length < 1) {
+        return new GraphqlData(GraphqlOperationType.QUERY);
+    }
+
+    let operationType = queryArray[0];
+    let operationName = queryArray[1];
+    if (!Object.values(GraphqlOperationType).includes(operationType)) {
+        operationType = GraphqlOperationType.QUERY;
+        operationName = queryArray[0];
+    }
+    if (req.body['operationName']) {
```

Comment:
Here for example. We set the operationName to be queryArray[0] but we change it if there is operationName in the body?

We need to find a way to set it once to avoid conflicts

Reply:
Done